### ISSUE TITLE
Add download/db-construction size/time estimation module

### ISSUE THREAD

**Issue by kbshimmyo** [NCBI-Hackathons/STREAMclean#15]
There's a TODO comment to indicate where this gets plugged in.

**Comment by arvkevi** [comment]
@halekpetigo does fbc02260ce2240da4d166375d7940fd0fbb8ccf2 resolve this issue?

**Comment by halekpetigo** [comment]
Yes, if we include the taxosize python script in mapper_wrapper.sh at line 127 <# TODO: this is where we should plug in the download size/time estimator>, that should address this issue. The Python script provides the size of the genome. The time to download depends on the network of the user hence difficult to estimate. Maybe we can provide a rough order of magnitude if we want to include the time.
